CarAccidentScenario/CarAccidentHandler.java

```diff
@@ -0,0 +1,67 @@
+package CarAccidentScenario;
+
+public class CarAccidentHandler {
+    public static void main(String[] args) {
+        try {
```

Comment:
Instead of having multiple try-catch block, which is basically calling all the exceptions, I think it's better to have one try- multiple catch- one finally blocks so it can simulate more real-life scenario.

Reply:
I have used this to avoid if else and switch case statements.

Reply:
if-else or switch-case will be used to simulate a car accident, so if we were to implement it for real then we would not be throwing any exceptions but rather they will come directly at run time in some method. But the try-catch block we are implementing here will stay the same even then, so I think it is a better approach to have a single try and multiple catch and maybe a finally block, instead of throwing all the exceptions here.